pandas/tests/io/parser/test_textreader.py

```diff
@@ -161,9 +161,9 @@ def test_skip_bad_lines(self):
                             error_bad_lines=False,
                             warn_bad_lines=False)
         result = reader.read()
-        expected = {0: ['a', 'd', 'g', 'l'],
-                    1: ['b', 'e', 'h', 'm'],
-                    2: ['c', 'f', 'i', 'n']}
+        expected = {0: np.array(['a', 'd', 'g', 'l'], dtype=object),
```

Comment:
I think assert can do this; don't convert to a numpy array just to compare

Reply:
The output of `Textreader.read` is an array, so would need to convert.